src/main/java/uk/gov/companieshouse/filinghistory/api/model/FilingHistoryData.java

```diff
@@ -0,0 +1,163 @@
+package uk.gov.companieshouse.filinghistory.api.model;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Objects;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class FilingHistoryData {
```

Comment:
Could this be a `record` instead?

Reply:
We would need to create a builder within it as the unit tests build partial objects. Would that be preferable?

Reply:
Let's leave it for now and revit. Adding a Builder to a record is simple